app/views/videos/index.html.haml

```diff
@@ -5,4 +5,4 @@
     .videos.row
       - category.recent_videos.each do |video|
         .video.col-sm-2
-          %img(src="#{video.small_cover_url}")
+          =link_to image_tag( src="#{video.small_cover_url}" ), video_path(video)
```

Comment:
`image_tag` can be written like this `image_tag video.small_cover_url` and you do not have to use full named routes like that. providing a video object is enough.